uber/templates/base.html

```diff
@@ -93,7 +93,13 @@
             if (message.length) {
                 toastr.info(message);
             }
```

Comment:
Huh, is ``DataTable`` really not case sensitive?  I see that you just swapped from upper to lower case, which I assumed was a typo, but I just tried running ``$('.foo').dataTable()`` on a page in production in my Javascript console and it didn't give me an error, so I assume it's also valid.

If so then +1 on this PR.